scripts/find_outliers.py

```diff
@@ -22,6 +26,196 @@ def find_outliers(data_directory):
     None
     """
     # Your code here
+
```

Comment:
You've put all your outlier detection code into one big function.  As we'll see later, this makes the code a lot harder to test, and it's hard to try different combinations of measures and detectors, because, as you've found, you have to do lots of commenting out blocks of stuff in the code, rather than doing things like:

```
outliers = iqr_detector(calc_dvars(data))
```

or 

```
outliers = sd_detector(some_other_measure(img_data))
```